src/frontends/tensorflow/tests/test_models/models_pbtxt/partitioned_call_and_if.pbtxt

```diff
@@ -0,0 +1,368 @@
+node {
```

Comment:
Same here. Model is called `partitioned_call` but no `PartitionedCall` operation in the graph

Reply:
Agree, in this case we have hard injection of the body without `PartitionCall`

Reply:
Renamed.